packages/salesforcedx-vscode-lwc/test/unit/commands/commandConstants.test.ts

```diff
@@ -10,10 +10,4 @@ describe('force:lightning:lwc:start constants', () => {
       expect(DEV_SERVER_BASE_URL).to.include('localhost');
     });
   });
-
-  describe('preview route', () => {
-    it('should include the base url', async () => {
-      expect(DEV_SERVER_PREVIEW_ROUTE).to.include(DEV_SERVER_BASE_URL);
```

Comment:
Instead of removing this test, could we change it to expect that the pattern matches DEV_SERVER_BASE_URL_REGEX?

Reply:
I'm now adding these regex tests in devServerService.test.ts